src/searchForm/SearchForm.story.mdx

```diff
@@ -0,0 +1,199 @@
+import { Meta, Preview, Props, Story } from '@storybook/addon-docs/blocks'
+import { text } from '@storybook/addon-knobs'
+
+import { TransitionDuration } from '../_utils/branding'
+import { MediaSizeProvider } from '../_utils/mediaSizeProvider'
+import { knobEnum } from '../../.storybook/knobs'
+import { AutoCompleteExample } from '../autoComplete/AutoCompleteExample'
+import { DatePicker } from '../datePicker'
+import { BaseSection, SectionContentSize } from '../layout/section/baseSection'
+import { AutoCompleteOverlay } from './autoComplete/overlay'
+import { AutoCompleteSection } from './autoComplete/section'
+import { DatePickerOverlay } from './datePicker/overlay'
+import { DatePickerSection } from './datePicker/section'
+import { SearchForm } from './index'
+import { SlideSwitchTransition, SlideSwitchTransitionSide } from './SlideSwitchTransition'
+import { StepperOverlay } from './stepper/overlay'
+import { StepperSection } from './stepper/section'
+
+<Meta title="Widgets|SearchForm" />
```

Comment:
I would suggest that you split this story on stories and keep only the "SearchForm" here

Reply:
Following @florianeyzat s comment in the description, I won't be updating this part:
> I tried creating a sub-story for each component to organize a bit and it worked BUT the generated doc was broken into pieces so I removed that

Reply:
I removed the other stories for some reasons we couldn't scroll on the docs page. The other stories were good for dev when we were working on those component, but since they're search form internals they shouldn't have their own stories